tests/Unit/Core/Parser/SyntaxTree/Expression/MathExpressionNodeTest.php

```diff
@@ -27,6 +27,14 @@ public static function getEvaluateExpressionTestValues(): array
             ['2 % 4', [], 2],
             ['2 * 4', [], 8],
             ['4 / 2', [], 2],
+            ['4 / 0.5', [], 8],
+            ['4 / 0', [], 0],
+            ['4 / 0.0', [], 0],
```

Comment:
Does Fluid support octal, binary or hex?
What would happen with `'4 / 0b0'`?

* hex: 0x0
* octal: 0o0
* binary 0b0

Reply:
Nope, it does not. All of them end up as float(0).

```
$result = '0x0';
var_dump(($result == (int)$result) ? (int)$result : (float)$result); // float(0)

$result = '0xa';
var_dump(hexdec($result)); // int(10)
var_dump(($result == (int)$result) ? (int)$result : (float)$result); // float(0)

$result = '0o0';
var_dump(($result == (int)$result) ? (int)$result : (float)$result); // float(0)

$result = '0b0';
var_dump(($result == (int)$result) ? (int)$result : (float)$result); // float(0)

```